tensorbay/opendataset/OxfordIIITPet/loader.py

```diff
@@ -78,7 +78,11 @@ def OxfordIIITPet(path: str) -> Dataset:
 
 def _get_box_label(file_path: str) -> List[LabeledBox2D]:
     with open(file_path, "r", encoding="utf-8") as fp:
-        objects = xmltodict.parse(fp.read())["annotation"]["object"]
+        labels = xmltodict.parse(fp.read())
+    if labels is None:
+        return []
```

Comment:
shall we raise exception rather than return here?

Reply:
If the xml file is empty, xmltodict will raise `xml.parsers.expat.ExpatError` itself.